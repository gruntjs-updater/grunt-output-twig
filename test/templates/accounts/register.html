{% extends "_layout" %}
{% set title = "Create an Account" %}
{% include "_includes/forms" %}

{% block content %}
	<form class="form" method="post" action="" accept-charset="UTF-8">
		<input type="hidden" name="action" value="users/saveUser">
		<input type="hidden" name="redirect" value="">

		<h3 class="first"><label for="username">Username</label></h3>
		<input id="username" class="text fulltext" type="text" name="username"
			{% if account is defined %} value="{{ account.username }}"{% endif %}>

		{% if account is defined %}
			Hi Account
		{% endif %}

		<h3><label for="email">Email</label></h3>
		<input id="email" class="text fulltext" type="text" name="email"
			{% if account is defined %} value="{{ account.email }}"{% endif %}>

		{% if account is defined %}
			Hi Account
		{% endif %}

		<h3><label for="password">Password</label></h3>
		<input id="password" class="text fulltext" type="password" name="password">

		<div class="buttons">
			<input class="submit" type="submit" value="Register">
		</div>
	</form>
{% endblock %}
