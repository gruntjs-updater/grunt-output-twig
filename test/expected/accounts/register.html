<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head>
			<meta charset="utf-8" />
		<title>Create an Account - On the Rocks</title>
		<meta name="viewport" content="width=1000" />
		<link rel="home" href="" />
		<link rel="stylesheet" type="text/css" href="/resources/css/ontherocks.css" />
	</head>

<body>
	<header id="header">
		<h1><a href="">On the Rocks</a></h1>

		<nav>
			<ul>
				<li><a  href="cocktails">Cocktails</a></li>
				<li><a  href="ingredients">Ingredients</a></li>
				<li><a  href="blog">Blog</a></li>
			</ul>
		</nav>

		<ul id="account">
							<li><a id="loginLink" href="">Login</a></li>
				<li><a href="accounts/register">Register</a></li>
					</ul>
	</header>

	<div id="main">
		
		<header>
			<h1>Create an Account</h1>
		</header>

		<div id="content">
										<form class="form" method="post" action="" accept-charset="UTF-8">
		<input type="hidden" name="action" value="users/saveUser">
		<input type="hidden" name="redirect" value="">

		<h3 class="first"><label for="username">Username</label></h3>
		<input id="username" class="text fulltext" type="text" name="username"
			 value="user">

					Hi Account
		
		<h3><label for="email">Email</label></h3>
		<input id="email" class="text fulltext" type="text" name="email"
			 value="email">

					Hi Account
		
		<h3><label for="password">Password</label></h3>
		<input id="password" class="text fulltext" type="password" name="password">

		<div class="buttons">
			<input class="submit" type="submit" value="Register">
		</div>
	</form>
		</div>
	</div>

	<script type="text/javascript" src="/resources/lib/jquery-1.8.2.min.js"></script>
	<script type="text/javascript" src="/resources/lib/packery.pkgd.min.js"></script>
	<script type="text/javascript" src="/resources/js/ontherocks.js"></script>
</body>
</html>
